vnext/ReactUWP/Views/FrameworkElementViewManager.cpp

```diff
@@ -129,6 +129,15 @@ void FrameworkElementViewManager::TransferProperties(
   auto tooltip = winrt::ToolTipService::GetToolTip(oldView);
   oldView.ClearValue(winrt::ToolTipService::ToolTipProperty());
   winrt::ToolTipService::SetToolTip(newView, tooltip);
+
+  // Clear the TransformMatrix from the old View.  The TransformMatrix will be
+  // set on the new View a bit later in RefreshProperties() (as we need data
+  // from the ShadowNode not available here).
+  auto oldElement = oldView.try_as<winrt::UIElement>();
+  if (oldElement && oldElement.try_as<winrt::IUIElement10>()) {
```

Comment:
Could this be simplified to this?
`if (auto oldElement = oldView.try_as<winrt::IUIElement10>()) {`

Reply:
This check is needed...  TransformMatrix is on IUIElement9 and our interfaces don't inherit.  We need to explicitly check IUIElement10 this has the ActualSize property, which is used in conjunction with TransformMatrix (used for centering the transform).  In other words we don't apply TransformMatrix unless we also have ActualSize available.